lua/plugins/lsp/codecompanion.lua

```
@@ -0,0 +1,76 @@
+function load_prompts()
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'load_prompts'

(W111)